types/winston-mail/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
```

Comment:
You shouldn't need this file, and definitely shouldn't have `private: true` in it

Reply:
Without the file the test fails in Travis. Any tips for another way to define the dependency for Winston?

Reply:
Sorry, didn't see that `winston` provides its own types. You should add a dependency on the `winston` package rather than `@types/winston`; see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d3af951dc97f7acdab00589b0cd000fbf4f05b2e/types/redux-storage/package.json for an example

Reply:
Doesn't seem like winston provides it's own types. Only way i can get the tests to pass without throwing missing type exports errors is by using `@types/winston `.